doc/modules/model_evaluation.rst

```diff
@@ -1382,7 +1382,7 @@ uniformly:
 
 .. math::
 
-   \frac{2}{c(c-1)}\sum_{j=1}^{c}\sum_{k > j}^c (\text{AUC}(j | k) +
+   \frac{1}{c(c-1)}\sum_{j=1}^{c}\sum_{k > j}^c (\text{AUC}(j | k) +
```

Comment:
For reviewers: This 2 should have been canceled out with the 2 used to average the two AUCs.